app/models/user.rb

```diff
@@ -9,4 +9,9 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
```

Comment:
- [ ] As per the ERD diagram, the Like table has a foreign key AuthorID, I, therefore, think that the association between user and likes should rather be:
`has_many :likes, foreign_key: 'author_id'`